processor/k8sattributesprocessor/internal/kube/client.go

```diff
@@ -25,6 +26,13 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sattributesprocessor/internal/observability"
 )
 
+// Upgrade to StageBeta in v0.83.0
+var enableRFC3339Timestamp = featuregate.GlobalRegistry().MustRegister(
+	"k8sattr.rfc3339",
+	featuregate.StageAlpha,
```

Comment:
We are bad about enforcing this, but since I remembered can you add a `featuregate.WithRegisterFromVersion()`

Reply:
done